apps/emqx_management/src/emqx_management.appup.src

```diff
@@ -1,6 +1,9 @@
-%% -*-: erlang -*-
+%% -*- mode: erlang -*-
 {VSN,
- [ {"4.3.2",
+ [ {"4.3.4",
+    [ {load_module, emqx_mgmt_api_data, brutal_purge, soft_purge, []}
+    ]},
+   {"4.3.2",
```

Comment:
add {load_module, emqx_mgmt_api_data, brutal_purge, soft_purge, []} to 4.3.2
Appup defines an upgrade from 4.3.2 to 4.3.3 (VSN)

{VSN, [upgrade], [downgrade]}

So you also need to add it in the list below

Reply:
app 4.3.2 is released in emqx release 4.3.2.
so it must be 4.3.3 this time.

Reply:
sorry, wrong thread to comment.